packages/design-tokens/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const postcssPresetEnv = require('postcss-preset-env');
+
+module.exports = (env, argv) => {
+	const isDev = argv.mode === 'development';
+	return {
+		mode: isDev ? 'development' : 'production',
+		entry: {
+			TalendDesignTokens: ['./src/index.ts', './src/index.scss'],
+		},
+		output: {
+			filename: '[name].js',
```

Comment:
depend on --dev or --prod, quite dangerous, I would tend to add a .min.js in prod and point to it in the package.json main